2-api/limits.md

```diff
@@ -0,0 +1,77 @@
+[Perspective API documentation](../README.md) > [API Reference Docs](README.md) > **LImits**
+
+# Limits and errors
+
+## Quota limit
+
+By default, we set the quota to 1 QPS for all Perspective projects.
+
+Check your quota limits by going to [your google cloud project's Perspective API page](https://console.cloud.google.com/apis/api/commentanalyzer.googleapis.com/quotas), and check your project's quota usage at
+[the cloud console quota usage page](https://console.cloud.google.com/iam-admin/quotas).
+
+Visit the Perspective API Support page to [request a quota increase](https://support.perspectiveapi.com/s/request-quota-increase).
+
+## Character limit for requests
+
+The maximum text size per request is 3000 bytes.
+
+## Error messages
+
+When using Outline, you might see a variety of error messages. This article details what each of those error messages might mean. 
```

Comment:
"Outline" is a separate product - did this info appear in the Support site for Perspective?

Reply:
It did indeed, so it definitely doesn't make sense.